configs/pie-components-config/placeholder.ts

```diff
@@ -0,0 +1 @@
+// This file exists solely to suppress the "No inputs were found in configuration file." TS warning.
```

Comment:
nit: do you think we can have another workaround that doesn't require creating an empty file? 

I assume we will eventually have a ts file somewhere in the directory to get rid of the issue, but maybe we can change the file extension of playwright files to ts? I don't think it will break anything but worth trying :D

Reply:
I was experimenting with your idea and came to the conclusion that it might not be as convenient as thought.

If we rename the playwright files to ts, the package index file also should be ts. With that, we lose the possibility of consuming `pie-components-config` files from vanilla js workspaces.

Reply:
The issue happens in the consuming side, if it's not a TS project it will reject the `index` file.

`TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts"`